Yoast/wordpress-seo
# 10527 replace snippet preview help button with text

## Summary

This PR can be summarized in the following changelog entry:

* Replaces the SnippetPreview help button with an explanation text above the snippet preview.

## Relevant technical choices:

* The snippet preview help button was removed in https://github.com/Yoast/yoast-components/pull/697
* Also removes unused component.

## Test instructions

This PR can be tested by following these steps:

* Make sure `yoast-components` is linked and https://github.com/Yoast/yoast-components/pull/697 is checked out.

## Quality assurance


Fixes #10527
